pajbot/managers/connection.py

```diff
@@ -112,15 +125,25 @@ def on_disconnect(self, _chatconn):
         log.error("Disconnected from IRC")
         self.start()
 
-    def privmsg(self, channel, message, increase_message=True):
+    def privmsg(self, channel, message, increase_message=True, whisper=False):
```

Comment:
I would prefer if you didn't introduce a `whisper` parameter and instead did the limiting logic in the `whisper` method

Reply:
How do we know if the message is a whisper?